maxim/logger/llamaindex/utils.py

```
@@ -0,0 +1,83 @@
+            raise ValueError(f"Unsupported message role: {message.role}")
+            
+    @staticmethod
+    def parse_message_to_generation_result(
+        message: ChatMessage
+        ) -> GenerationResultMessage:
+        
+        parsed_messages = GenerationResultMessage(role=message.role, content=parse_message_content_to_generation_result(message.blocks), tool_calls=[])
+        
+        return parsed_messages
+
+def parse_message_content(blocks: List[ContentBlock]):
```

Ruff (0.12.2)

Missing return type annotation for public function `parse_message_content`

(ANN201)